### PR TITLE
[DOCFIX] Update metrics doc update for fuse

### PR DESCRIPTION
**Pull request by jja725**
### What changes are proposed in this pull request?

update access to fuse metrics

### Why are the changes needed?

Please clarify why the changes are needed. For instance,
  1. If you propose a new API, clarify the use case for a new API.
  2. If you fix a bug, describe the bug.

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including
  1. change in user-facing APIs
  2. addition or removal of property keys
  3. webui

### CODE REVIEW

File: docs/en/operation/Metrics-System.md [hunk 1]
```diff
@@ -65,6 +65,16 @@ $ curl <WORKER_HOSTNAME>:<WORKER_WEB_PORT>/metrics/json
 $ curl 127.0.0.1:30000/metrics/json/
 ``` 
 
+Send an HTTP request to `/metrics/json/` to fuse web port to get fuse metrics.
```
Comment (LuQQiu) [inline docs/en/operation/Metrics-System.md:72]: I synced with @ssz1997 about updating the Alluxio POSIX API about enabling the fuse web in standalone fuse process.

After he created the PR and merged, can you help link to that doc and mentioning
if alluxio fuse web port is enabled by alluxio.fuse.web.enabled=true, send an HTTP request to `/metrics/json/` to fuse web port to get fuse metrics.

Comment (jja725) [reply docs/en/operation/Metrics-System.md]: @LuQQiu should be good now, PTAL

File: docs/en/operation/Metrics-System.md [hunk 2]
```diff
@@ -67,6 +67,26 @@ $ curl <WORKER_HOSTNAME>:<WORKER_WEB_PORT>/metrics/json/
 $ curl 127.0.0.1:19999/metrics/json/
 # Get the local worker metrics with its default web port 30000
 $ curl 127.0.0.1:30000/metrics/json/
+``` 
+docs/en/api/POSIX-API.md
```
Comment (LuQQiu) [inline docs/en/operation/Metrics-System.md:71]: ?? this line can be removed?

**Review by LuQQiu — APPROVED** [review]
LGTM except an extra line

File: docs/en/operation/Metrics-System.md (hunk 2 above)
Comment (jja725) [reply docs/en/operation/Metrics-System.md]: fixed

File: docs/en/operation/Metrics-System.md [hunk 3]
```diff
@@ -67,6 +67,26 @@ $ curl <WORKER_HOSTNAME>:<WORKER_WEB_PORT>/metrics/json/
...
+
+```console
+# Get the metrics in JSON format from an active Alluxio FUSE process.
+$ curl <FUSE_HOSTNAME>:<FUSE_WEB_PORT>/metrics/json
+
+# For example, get the metrics from fuse process running locally with default web port
+$ curl 127.0.0.1:49999/metrics/json/
+``` 
+
+### CSV Sink Setup
```
Comment (LuQQiu) [inline docs/en/operation/Metrics-System.md:82]: @jja725 is this section needed?
looks like it's not the full CSV sink setup
isn't it in the doc already?

Comment (jja725) [reply docs/en/operation/Metrics-System.md]: might be some problem when merging master. Thanks for finding this and Fixed

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
@Xenorith cherry-pick?

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Pull request merged**